### DISCUSSION TITLE
Dialog editor

### DISCUSSION THREAD

**Discussion by joegittingherdone** [open-watcom/open-watcom-v2#686] [Q&A] [answered]
Hello,
When I run the Watcom Dialog Editor (wde.exe) in Windows 7 (64-bit) and then try to open bld/vi/res/ssbar.dlg, I get an error: "Resources not loaded."  Do I need Microsoft software to edit this?

**Comment by jmalak** [comment, marked as answer]
I thing it is misunderstanding.
Dialog editor works with compiled dialog resource not with source code.
bld/vi/res/ssbar.dlg is source text form for resource compiler.
In this case open compiled viw.exe file by dialog editor and select appropriate dialog.
For editing ssbar.dlg source file you need to understand "resource compiler language".

**Reply by jmalak** [reply]
The OW dialog editor is capable to output source file for dialog.
You can use this source file for building resources by resource compiler or you can create/edit dialog in resource binary form and add it to executable as additional resource.

**Comment by joegittingherdone** [comment]
I understand; thank you.
